src/music/dmusic.cpp

```diff
@@ -972,7 +972,7 @@ static std::optional<std::string_view> LoadDefaultDLSFile(std::optional<std::str
 				download_port->Release();
 				return "Can't get instrument download buffer";
 			}
-			char *inst_base = (char *)instrument;
+			const std::byte *inst_base = reinterpret_cast<const std::byte *>(instrument);
```

Comment:
Perhaps `instrument` could be typed as `byte*` anyway, given it's already getting casted to `LPVOID*` on use ? Might save a cast or two, not sure

Reply:
To be honest that whole bit of code is a mess, but it's not something I want to tackle via building through GitHub.